guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/ProjectResourcePaths.java

```diff
@@ -0,0 +1,21 @@
+package org.guvnor.common.services.project.backend.server;
```

Comment:
I tried to setup a check like that years ago, but at that time the tooling was not ready. But it might work today, if not for malformed, then just for missing headers.